packages/builder/src/services/pinata.ts

```diff
@@ -27,7 +27,11 @@ export default class PinataClient {
   }
 
   fileURL(cid: string) {
-    return `${GATEWAY}/ipfs/${cid}`;
+    // if GATEWAY is true it throws an exception in the constructor
+    const baseURL = new URL(GATEWAY!);
```

Comment:
The non-null assertion would not be needed here if `this.gateway` were used, since that has already been validated as being non-null in the constructor.

Reply:
@bard nice catch thank you! in fact I added the comment but I then used the other variable instead of the instance one thank you!

Reply:
I updated it using `const baseURL = new URL(this.gateway);`